npm/cypress-schematic/sandbox13/angular.json

```diff
@@ -3,7 +3,7 @@
   "version": 1,
   "newProjectRoot": "projects",
```

Comment:
I wonder if we can reuse the projects in `system-tests` so we don't need to add so many test projects?

![image](https://user-images.githubusercontent.com/19196536/180883496-0aa289f1-e8ab-4ba6-824c-001fd0912de3.png)